shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -187,6 +187,10 @@
  *     android:value="true"
  *     />
  * }
+ * <p>
+ * A number of methods in this class have the same implementation as
```

Comment:
Should these be javadocs? I don't think people want to see it in https://api.flutter.dev/javadoc/ right? Maybe split this into 2 parts. Reference the other alternative in javadocs for users and leave a code comment for other maintainers.

Reply:
Sounds good. Made the changes.

Reply:
LG